src/stories/molecules/tabs/tabs.stories.html

```diff
@@ -0,0 +1,24 @@
+<section class="stories__molecules">
```

Comment:
extract the content to json, or is this part of the wip

Reply:
Nice catch, you are right ;-)

Reply:
Didn't really realize yesterday that the content can only be added through the default component slot. So there is no option use json as input.